exercises/4_4/ParteI/ex05.js

```diff
@@ -0,0 +1,17 @@
+let info = {
+  personagem: "Margarida",
+  origem: "Pato Donald",
+  nota: "Namorada do personagem principal nos quadrinhos do Pato Donald",
+};
+
+info.recorrente = "sim";
+
+let info2 = Object.create(info);
+
+info2.personagem = "Tio Patinhas";
+info2.origem = "Christmas on Bear Mountain, Dell’s Four Color Comics #178";
+info2.origem += "O último MacPatinhas";
+
+for(key in info){
+  console.log(info[key] + " e " +info2[key]);
```

Comment:
Luiz, não sei se já viu, mas um jeito legal de concatenar também é usando string interpolation. Aqui tem umas informações legais sobre isso: https://medium.com/@endubueze00/javascript-basics-string-concatenation-with-variables-and-interpolation-deba239debbe